doc/source/changes/version_0_21.rst.inc

```diff
@@ -0,0 +1,110 @@
+﻿New features
+------------
+
+* implemented replaced_axes() method to replace one, several or all axes of an array
+  (instead of using the workaround a.with_axes(a.axes.replace(...)))
+  (closes :issue:`67`).
+
+    >>> arr = ndtest((2, 3))
+    >>> arr
+    a\\b | b0 | b1 | b2
+     a0 |  0 |  1 |  2
+     a1 |  3 |  4 |  5
+    >>> row = Axis('row', ['r0', 'r1'])
+    >>> column = Axis('column', ['c0', 'c1', 'c2'])
+
+    Replace one axis (expect 2 arguments)
+
+    >>> arr.replace_axes(x.a, row)
+    row\\b | b0 | b1 | b2
+       r0 |  0 |  1 |  2
+       r1 |  3 |  4 |  5
+
+    Replace several axes (keywords, list of tuple or dictionary)
+
+    >>> arr.replace_axes(a=row, b=column)
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+    >>> arr.replace_axes([(x.a, row), (x.b, column)])
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+    >>> arr.replace_axes({x.a: row, x.b: column})
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+
+    Replace all axes (list of axes or AxisCollection)
+
+    >>> arr.replace_axes([row, column])
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+    >>> arr2 = ndrange([row, column])
+    >>> arr.replace_axes(arr2.axes)
+    row\\column | c0 | c1 | c2
+            r0 |  0 |  1 |  2
+            r1 |  3 |  4 |  5
+
+* implemented from_string() method to create an array from a string:
+
+    >>> from_string('''nat\\sex, M, F
+    ...                BE,       0, 1
+    ...                FO,       2, 3''')
+    nat\sex | M | F
+         BE | 0 | 1
+         FO | 2 | 3
+    >>> from_string('''age,nat\\sex, M, F
```

Comment:
Isn't a single example enough to get the point across? Unsure whether we need the 3D or 2D one.